cpp/conv_templates.cc

```diff
@@ -385,10 +385,10 @@ Conversation CodeGPT() {
   return conv;
 }
 
-Conversation WizardLM() {
+Conversation WizardLM7B() {
```

Comment:
Why we need to specify "7B"? Will Wizard uses different prompt for different model size?

Reply:
Yep, 7B does not support multi-round chat, while all other sizes do. See more at: https://github.com/nlpxucan/WizardLM/blob/main/WizardLM/README.md